src/singletons/NativeMessaging.cpp

```diff
@@ -263,6 +279,39 @@ void NativeMessagingServer::ReceiverThread::handleDetach(
 }
 // NOLINTEND(readability-convert-member-functions-to-static)
 
+void NativeMessagingServer::ReceiverThread::handleSync(const QJsonObject &root)
+{
+    // Structure:
+    // { action: 'sync', twitch?: string[] }
```

Comment:
Let's be explicit with what's being synced, rename the `twitch` key to `twitchChannels` or `twitch_channels` (whichever standard is used in the other json fields)